lib/helpers/parameter-store.js

```
@@ -0,0 +1,103 @@
+	static async getParameterArnFromRAM() {
+
+		try {
+
+			const command = new ListResourcesCommand({ resourceOwner: 'OTHER-ACCOUNTS' });
+
+			const ramClient = new RAMClient({ region: 'us-east-1' });
+
+			const response = await ramClient.send(command);
+
+			const filteredResources = response.resources.filter(
+				resource => resource.arn.includes(this.parameterName)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)